CRM/Event/BAO/Event.php

```diff
@@ -186,16 +186,26 @@ public static function del($id) {
     $extends = array('event');
     $groupTree = CRM_Core_BAO_CustomGroup::getGroupDetail(NULL, NULL, $extends);
     foreach ($groupTree as $values) {
-      $query = "DELETE FROM " . $values['table_name'] . " WHERE entity_id = " . $id;
-
-      $params = array(
-        1 => array($values['table_name'], 'string'),
-        2 => array($id, 'integer'),
-      );
-
-      CRM_Core_DAO::executeQuery($query);
+      $query = "DELETE FROM " . $values['table_name'] . " WHERE entity_id = %1";
```

Comment:
This is the pattern used elsewhere. We cannot call the table name a 'string' parameter, because it must not be enclosed in single quotes - otherwise the SQL is invalid. But we are checking the entity ID now.